dev-libs/newt/newt-0.52.21-r1.ebuild

```diff
@@ -23,7 +23,6 @@ RDEPEND="
 	${PYTHON_DEPS}
 	>=dev-libs/popt-1.6
 	=sys-libs/slang-2*
-	elibc_uclibc? ( sys-libs/ncurses:0= )
```

Comment:
It _really_ only needs ncurses on uclibc?

Reply:
I see no sign of curses being used in that package *anywhere*, I haven't checked whether it calls one of the executables, but it definitely doesn't link to curses